vars/createArgusTestRun.groovy

```diff
@@ -16,4 +16,8 @@ def call(Map params) {
 
         echo " Argus test run created."
     """
+    if (!currentBuild.description) {
+        currentBuild.description = ''
+    }
+    currentBuild.description += "\n<a href='https://argus.scylladb.com/tests/scylla-cluster-tests/${SCT_TEST_ID}'>Argus</a>\n"
```

Comment:
we should add the test id, or part of it into the test
in case there are multiple test ids in one job (and we do have those)

Reply:
Done.. with a small improvement to the visibility.

![image](https://github.com/user-attachments/assets/387d3855-a66b-4d13-85d7-2b4688df94f8)

Reply:
However there is a small issue in the build sidebar:
![image](https://github.com/user-attachments/assets/af7aaee6-75b8-456e-b040-a0eeb6b1a4b6)

Reply:
Okay, fixed with text-overflow, unfortunately it cuts all text in the sidebar:
![image](https://github.com/user-attachments/assets/02899eda-5f1b-4c2d-afac-6f4d73582f55)

Reply:
![image](https://github.com/user-attachments/assets/c1a1f6fd-95c9-4b32-996c-241a14363572)
![image](https://github.com/user-attachments/assets/fe67d0c3-0736-477c-976e-ef29d0417794)

Final-ish look